components/brave_rewards/browser/database/publisher_info_database.cc

```diff
@@ -1132,6 +1129,57 @@ ledger::ServerPublisherInfoPtr PublisherInfoDatabase::GetServerPublisherInfo(
 
 // Other -------------------------------------------------------------------
 
+void PublisherInfoDatabase::RecordP3AStats(bool auto_contributions_on) {
+  if (!initialized_) {
+    return;
+  }
+
+  sql::Statement sql(
+      db_.GetCachedStatement(SQL_FROM_HERE,
+                             "SELECT category, COUNT(*) FROM contribution_info "
+                             "WHERE category in (?, ?, ?) GROUP BY 1"));
+
+  sql.BindInt(0, ledger::AUTO_CONTRIBUTE);
```

Comment:
not sure if this is needed as it's always one of those. I would say that we can remove `WHERE` portion in the query

Reply:
fixed